.github/workflows/build.yml

```diff
@@ -0,0 +1,26 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+        java_version:
+          - 11
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java_version }}
```

Comment:
Can we just set up all the Java-Versions from 7-14 here and then choose a specific one for the Gradle build? E.g. `JAVA_HOME=${howToFindOut?} gradlew build`?

Reply:
Hmmm, what benefit do you expect from this solution? I think it is easier to let the matrix build take care of these env vars and use it accordingly rather than using it as a kind of "option provider that requires manual interaction".

Reply:
Ah, I see. You refer to the `runMavenIntegrationTest` task. Here, I thought about publishing each successful build into the Github private registry and then use the real artifact for tests with different JDKs. But I am still investigating whether this is easily possible as it might require some refactoring of the test run.

Reply:
Ah, okay, yes, this makes sense :+1: It will be the real `publish` task then to publish to the GitHub registry? That would give an additional level of testing which would be really neat :smiley: